packages/myst-directives/src/code.ts

```diff
@@ -209,15 +209,15 @@ export const codeCellDirective: DirectiveSpec = {
       executable: true,
       value: (data.body ?? '') as string,
     };
-    const output = {
-      type: 'output',
+    const outputs = {
+      type: 'outputs',
       id: nanoid(),
```

Comment:
@agoose77 adding this immediately enables compute in figures & embeds in many of the test cases. If there is a place in `myst-execute` where this should be added I suggest we do. If you could please do that, or if it's trivial, point me to the line to change.

Reply:
We need this change. on this PR https://github.com/jupyter-book/myst-theme/pull/571 you mentioned that `myst-execute` does not add the id, so I'm just flagging that we need one. Adding the id here seems to be enough, but if there is a further change needed in `myst-execute`, in line with your comment that should also be changed.

Reply:
Ah I see!